components/src/ProductRecommendations/productrecommendations.main.stories.tsx

```diff
@@ -1,12 +1,13 @@
 import React from 'react';
 import { storiesOf } from '@storybook/react';
 import { MemoryRouter } from 'react-router';
-import productData from './MockHttpResponses/bundle_constituents_response.json';
+import crosssellProductData from './MockHttpResponses/GET/crosssell_response.json';
 
 import ProductRecommendationsDisplayMain from './productrecommendations.main';
 
 storiesOf('ProductRecommendationsDisplayMain', module)
   .addDecorator(story => (
     <MemoryRouter initialEntries={['/']}>{story()}</MemoryRouter>
   ))
-  .add('ProductRecommendationsDisplayMain', () => <ProductRecommendationsDisplayMain productData={productData} />);
+  .add('ProductRecommendationsDisplayMain', () => <ProductRecommendationsDisplayMain productData={crosssellProductData} />);
+  
```

Comment:
Please fix the broken line break 😅

Reply:
updated! Also added some additional fixes